### ISSUE TITLE
작가님~ 폴더 설정에 관해서 질문드려요

### ISSUE THREAD

**Issue by JungInkeun** [ratsgo/nlpbook#36]
제가 bert 모델로 학습한 결과 파일을 저장하는 경로로 책과 똑같이 이렇게 했습니다.
args = ClassificationTrainArguments(
    pretrained_model_name = 'beomi/kcbert-base',
    downstream_corpus_name = 'nsmc',
    downstream_model_dir='/gdrive/MyDrive/연구전략_파이썬공부/BERT와GPT로배우는자연어처리/checkpoint-doccls',
    batch_size = 32 if torch.cuda.is_available() else 4,
    learning_rate=5e-5,
    max_seq_length=128,
    epochs=1,
    tpu_cores=0 if torch.cuda.is_available() else 8,
    seed=7
)

그리고 학습을 시켰는데요 이런 메시지가 나오는데 ckpt로 저장된 파일은 안 보입니다.ㅡㅜ

LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
/usr/local/lib/python3.7/dist-packages/pytorch_lightning/core/optimizer.py:381: RuntimeWarning: Found unsupported keys in the optimizer configuration: {'scheduler'}
  f"Found unsupported keys in the optimizer configuration: {set(extra_keys)}", category=RuntimeWarning

혹시나 해서 Korpora 폴더의 nsmc 폴더에 갔는데 파일이 4개 있는데요 혹시 이게 그 체크포인트파일(ckpt) 인가요? 도와주세요ㅜㅡ

cached_test_BertTokenizer_128_nsmc_document-classification.lock
cached_test_BertTokenizer_128_nsmc_document-classification

**Comment by ratsgo** [comment]
안녕하세요 @JungInkeun 님 답변이 너무 늦었습니다.
제 환경에서는 해당 문제가 재현이 안되는데요ㅠㅠ 
(아마도 `downstream_model_dir`의 문제이지 않을까 싶긴 합니다만^^;)
혹시 다음 코랩 노트북을 어떤 코드의 수정도 없이 실행해 보시면 어떠실까요?

- https://colab.research.google.com/github/ratsgo/nlpbook/blob/master/examples/document_classification/train_colab.ipynb

실행해보시고 문제가 있다면 코랩 화면을 캡처해서 그대로 공유해주시면 감사하겠습니다.
